clusterloader2/testing/density/config.yaml

```diff
@@ -0,0 +1,114 @@
+automanagedNamespaces: 3
+tuningSets:
+- name: Uniform100qps
+  qpsLoad:
+    qps: 100
+- name: Uniform5qps
+  qpsLoad:
+    qps: 5
+steps:
+- measurements:
+  - Identifier: TimerGlobal
+    Method: Timer
+    Params:
+      action: start
+  - Identifier: TimerCreateSaturationPods
```

Comment:
nit: I'd suggest putting 'Timer' at the end of the name? (here and elsewhere)

Reply:
done.